test/puppetlabs/puppetdb/http/facts_test.clj

```diff
@@ -590,8 +591,8 @@
 (deftestseq ^{:postgres false} two-database-fact-query-config
   [[version endpoint] facts-endpoints]
 
-  (let [read-db-config (create-db-map)
-        write-db-config (create-db-map)
+  (let [read-db-config (create-hsqldb-map)
+        write-db-config (create-hsqldb-map)
```

Comment:
It get's skipped, according to the test-selector `^{:postgres false}`, see our project.clj